saleor/graphql/tax/mutations/tax_country_configuration_update.py

```diff
@@ -67,14 +68,44 @@ def clean_input(cls, **data):
         cleaned_data = {}
         failed_ids = []
 
+        # Check if only one default rate is provided (only one item without the tax
+        # class).
+        default_rate_items = [
+            item for item in update_tax_class_rates if item.get("tax_class_id") is None
+        ]
+        if len(default_rate_items) > 1:
+            code = (
+                error_codes.TaxCountryConfigurationUpdateErrorCode.ONLY_ONE_DEFAULT_COUNTRY_RATE_ALLOWED  # noqa: E501
+            )
+            params = {"tax_class_ids": []}
+            raise ValidationError(
+                {
+                    "update_tax_class_rates": ValidationError(
+                        code=code,
+                        message=(
+                            "Only one default country rate can be created for "
+                            "a country (a rate without a tax class)."
+                        ),
+                        params=params,
+                    )
+                }
+            )
+
+        # Clean IDs
         for item in update_tax_class_rates:
-            global_id = item["tax_class_id"]
-            try:
-                _, pk = from_global_id_or_error(global_id, "TaxClass", raise_error=True)
-            except GraphQLError:
-                failed_ids.append(global_id)
-            else:
-                cleaned_data[int(pk)] = item
+            global_id = item.get("tax_class_id")
+            pk = None
+            if global_id:
+                try:
+                    _, pk = from_global_id_or_error(
+                        global_id, "TaxClass", raise_error=True
+                    )
+                except GraphQLError:
+                    failed_ids.append(global_id)
+                    continue
+                pk = int(pk)
```

Comment:
IMO, instead of adding a comment, we can create a method from this.

Reply:
Fixed